contracts/contracts/token/WOETH.sol

```diff
@@ -66,6 +86,12 @@ contract WOETH is ERC4626, Governable, Initializable {
          * environment.
          */
         oethCreditsHighres = _getOETHCredits();
+        creditsPerTokenLimit = OETH(asset())
```

Comment:
Can we just remove this entire section of the initializer and call `increaseYieldLimit()`. This would deduplicate the code.

Reply:
Agreed thanks: https://github.com/OriginProtocol/origin-dollar/pull/2447/commits/821a75689d924644a00edbbba5f54c24eb071b65